packages/quantic/force-app/main/default/lwc/quanticHeading/quanticHeading.js

```diff
@@ -0,0 +1,31 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticHeading` component offers the ability to display a label with a customizable heading level.
+ * @category Utility
+ * @example
+ * <c-quantic-heading label="My label" level="1"></c-quantic-heading>
+ */
+export default class QuanticHeading extends LightningElement {
+  /**
+   * The label dsiplayed inside the heading.
+   * @api
+   * @type {string}
+   */
+  @api label;
+  /**
+   * The level to use for the heading label, accepted values are integers from 1 to 6.
+   * A value outside of the range of 1 to 6 will render a div instead of a heading.
+   * @type {number}
+   */
+  @api level;
+
+  renderedCallback() {
+    const heading = this.template.querySelector('[data-section="label"]');
+    // @ts-ignore
+    const level = parseInt(this.level, 10);
+    const headingTag = level > 0 && level <= 6 ? `h${level}` : 'div';
+    // eslint-disable-next-line @lwc/lwc/no-inner-html
+    heading.innerHTML = `<${headingTag}>${this.label}</h${headingTag}`;
```

Comment:
I recommend not to insert a prop like this with innerHTML, especially since it's not obvious from outside this component that `label` can be used to do an XSS injection.

Instead, I would recommend using `document.createElement` and `innerText`.

Reply:
also, there's an error here, the closing tag becomes `</h11>` for level=1

Reply:
Thanks for pointing these out! both issues have been fixed!